api.md

```diff
@@ -1034,6 +1035,43 @@ Create new databases `dist_database` on data nodes, which requires to set `trans
 CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 ```
 
+---
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a distributed database.
+
+This function is an analogue of `pg_create_restore_point()` designed to work with a distributed database
+and taking into account synchronization of the remote transactions.
+
```

Comment:
I think it is important to describe what it does:
- It will obtain locks and lock all distributed hypertables for queries to have a consistent state at the restore point.
- It will create restore point on every data node and the access node with the given name.

Reply:
I guess I can add more info about it, but just to note it does not lock any hypertables at all. It is more about remote transaction synchronization and postponing `COMMIT PREPARE` command in some cases to ensure consistency. Not really sure that this implementation details should be exposed here.

Reply:
Sorry, I believe I wasn't correct in the formulation. I feel it will be good for users to know that the distributed restore point is blocking, so uncommitted distributed transactions will wait until the function completes. Do I follow correctly?

Reply:
It does not block remote transactions completely, only when they decide it is time to commit. This commit operation might be postponed until the function completes, in order to achieve consistent WAL history across the cluster. I don't really think this should be mentioned in more details then necessary in the API description, unless people disagree.

Reply:
I see the improvement. I think it is good.